style.css/thirdsectionStyle.css

```diff
@@ -0,0 +1,113 @@
+.third-section {
+  text-align: center;
+}
+
+.title-third-section {
+  background-color: red;
+  color: #ffffff;
+  display: inline-block;
+  padding: 8px;
+  font-weight: bold;
+  margin-bottom: 64px;
+  margin-top: 40px;
+}
+
+.project-list {
+  padding: 0;
+  gap: 25px;
+  display: grid;
+  box-sizing: border-box;
+}
+
+.project-list-item {
+  flex: 50%;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+}
+
+.title-feature {
+  text-align: start;
+  color: red;
+  margin-top: 24px;
+  font-weight: bold;
+}
+
+.content-feature {
+  text-align: start;
+  margin-top: 16px;
+  margin-bottom: 8px;
+  line-height: 25px;
+}
+
+.link-list {
+  display: flex;
+  padding: 0;
+  gap: 10px;
+}
+
+.link-list-item {
+  background: #fddfe7;
+  padding: 2px 4px;
+  text-align: center;
+}
+
+.project-list-header {
+  text-align: center;
+  border: 1px solid red;
+  font-weight: bold;
+}
+
+.project-list-title {
+  text-align: center;
+  font-weight: bold;
+  color: red;
+  margin-bottom: 32px;
+  margin-top: 64px;
+}
+
+.app-feature {
+  text-align: start;
+  margin-top: 16px;
+  margin-bottom: 8px;
+  line-height: 25px;
+}
+
+.app-list {
+  display: flex;
+  justify-content: start;
+  gap: 15px;
+  padding: 0;
+}
+
+.app-list-item {
+  background: #fddfe7;
+}
+
+.title-app-feature {
```

Comment:
It's not aligned with the design. Title and p should be in the same line and also need to underline. Please check the design.

Reply:
you can use display flex for it. And if they are not align you need to check each margin and padding.